src/styles/technology.scss

```
@@ -0,0 +1,180 @@
+.technology {
+  max-width: 1200px;
+  overflow: hidden;
+
+  &__heading {
+    margin-top: 150px;
+    margin-left: 100px;
+  }
+
+  &__features {
+    position: relative;
+    height: 546px;
+    width: 1200px;
+
+    &::after {
+      content: "";
+      display: block;
+      position: absolute;
+      bottom: 12%;
+      left: 20%;
+      width: 700px;
+      height: 20px;
+      border-radius: 50%;
+      background-color: $main-bg-color;
+      transform: rotate(-5.14deg);
+      filter: drop-shadow(10px 100px 30px $color-gray);
+      z-index: -1;
+    }
+  }
+
+  &__img {
+    width: 1230px;
+    transform: rotate(-7.85deg);
+  }
+
+  &__part {
+    position: absolute;
+
+    width: 48px;
+    height: 48px;
+
+    background-color: $color-green;
+    border-radius: 50%;
+
+    color: white;
+    font-size: 22px;
+    text-align: center;
+    line-height: 48px;
+
+    transition: 300ms;
+
+    cursor: pointer;
+
+    &:hover {
+      background-color: #131313;
+    }
+
+    &1 {
+      left: 6%;
+      top: 40%;
+    }
+
+    &2 {
+      left: 20%;
+      top: 64%;
+    }
+
+    &3 {
+      left: 26%;
+      top: 52%;
+    }
+
+    &4 {
+      left: 32%;
+      top: 24%;
+    }
+
+    &5 {
+      left: 40%;
+      top: 74%;
+    }
+
+    &6 {
+      left: 51%;
+      top: 30%;
+    }
+
+    &7 {
+      left: 63%;
+      top: 55%;
+    }
+
+    &8 {
+      left: 75%;
+      top: 30%;
+    }
+
+    &9 {
+      left: 86%;
+      top: 13%;
+    }
+
+    &10--active {
+      left: 92%;
+      top: 36%;
+
+      background-color: #131313;
+
+      &::after {
+        content: "";
+        display: block;
+        height: 249px;
+        width: 200px;
+        background-color: transparent;
+        border-right: 1px black solid;
+        border-bottom: 1px black solid;
+        margin: -176px;
+        margin-top: 26px;
+      }
+    }
+
+    &11 {
+      left: 96%;
+      top: 26%;
+    }
+  }
+
+  &__message {
+    position: absolute;
+    left: 77%;
+    top: 85%;
+
+    max-width: 120px;
+
+    &::after {
+      content: "";
+      display: block;
+      width: 63px;
+      height: 5px;
+
+      margin-top: 10px;
+
+      background-color: $color-black;
+    }
+  }
+}
+
+@media (max-width: $tablet-screen) {
+  .technology {
+    &__heading {
+      margin-top: 130px;
+    }
+  }
+}
+
+@media (max-width: $phone-screen) {
+  .technology {
+    &__features {
+      max-width: 624px;
+      height: 277px;
+    }
+
+    &__img {
+      max-width: 624px;
+    }
+
+    &__heading {
+      margin-top: 0;
+      margin-left: 30px;
+    }
+
+    &__part--10 {
```

тут ты пишешь так, а до этого ты писал так
```suggestion
    &__part {
      &--10 {
```
нужно писать везде одинаково